### ISSUE TITLE
crack detection model question

### ISSUE THREAD

**Issue by gkseogus** [priya-dwivedi/Deep-Learning#142]
For pytorch 1.1 or later, skip the first value of the learning rate schedule optimizer.step() if you use the learning rate scheduler before updating the optimizer.step(). There is an error saying that. How do we deal with this?
